submissions/Roophee/a-tyni-js-world-roophee/index.js

```diff
@@ -0,0 +1,66 @@
+const dog = {
+    species: 'dog',
+    name: 'Milo',
+    gender: 'male',
+    legs: 4,
+    hands: 0,
+    tail: 1,
+    saying: 'Woof-Woof!',
+    friends: ['Sam', 'Hanna', 'Kitty']
+}
+
+const cat = {
+    species: 'cat',
+    name: 'Kitty',
+    gender: 'female',
+    legs: 4,
+    hands: 0,
+    tail: 1,
+    saying: 'Meow!',
+    friends: []
+}
+
+const man = {
+    species: 'human',
+    name: 'Sam',
+    gender: 'male',
+    legs: 2,
+    hands: 2,
+    tail: 0,
+    saying: 'Hello, my friend!',
+    friends: ['Hanna',]
+}
+
+const woman = {
+    species: 'human',
+    name: 'Hanna',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    tail: 0,
+    saying: 'Hi, I\`m Hanna!',
+    friends: ['Sam', 'Kitty', 'Milo']
+}
+
+const womanCat = {
+    species: 'human',
+    name: 'Sara',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    tail: 1,
+    saying: cat.saying,
+    friends: ['Sam', 'Hanna', 'Kitty', 'Milo']
+}
+
+let toStringTask = function (item) {
+    let string = '';
+    for (let i in item) {
+        string += item[i] + ';';
+    }
+    return string;
+}
```

Comment:
Please check yourself against requirements.

![image](https://user-images.githubusercontent.com/8686631/103037508-a879a780-4574-11eb-8567-94eb1e6f9ebb.png)

`for...in` is a variant of `Object.keys()`

Reply:
Sorry for this and thanks for your patience. 

Fix this. Rewrite method that create string.